assets/config/postcss.config.js

```diff
@@ -16,6 +16,34 @@ module.exports = {
             ]
         }),
 
+        // PurgeCSS to remove unused classes for better page speed.
+        // Docs: https://purgecss.com/plugins/postcss.html
+        require("@fullhuman/postcss-purgecss")({
+            // Specify the paths to all of the template files in your project
+            content: [
+                "./layouts/**/*.html",
+                // Some of our scripts reference CSS classes.
+                "./assets/js/**/*.js",
+                // Look for CSS classes in our markdown content. We use `glob` explicitly here so we can ignore the
+                // files for the API docs in ./content/docs/reference/pkg/**/* because it includes a large number of
+                // files that significantly impacts build time (~25 seconds vs. many minutes). Instead, we'll only look
+                // for CSS classes in a single package (the Pulumi SDK package) for nodejs and python, which should
+                // include all classes used in the docs for any other package.
+                ...require("glob").sync("./content/**/*.md", { ignore: "./content/docs/reference/pkg/**/*" }),
+                "./content/docs/reference/pkg/nodejs/pulumi/pulumi/**/*.md",
+                "./content/docs/reference/pkg/python/pulumi/**/*.md",
+            ],
+            // Whitelist HubSpot specific classes so they don't get removed.
```

Comment:
This is more than just HubSpot now, right?

Reply:
Yup, good catch. I updated the comments!